p2p/peersHolder/connectionStringValidator/connectionStringValidator_test.go

```diff
@@ -0,0 +1,54 @@
+package connectionStringValidator
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_connectionStringValidator_IsValid(t *testing.T) {
+	t.Parallel()
+
+	csv := NewConnectionStringValidator()
+	assert.False(t, csv.IsValid("invalid string"))
+
+	assert.True(t, csv.IsValid("5.22.219.242"))
+	assert.True(t, csv.IsValid("2031:0:130F:0:0:9C0:876A:130B"))
+	assert.True(t, csv.IsValid("16Uiu2HAm6yvbp1oZ6zjnWsn9FdRqBSaQkbhELyaThuq48ybdojvJ"))
+}
+func Test_connectionStringValidator_isValidIP(t *testing.T) {
```

Comment:
add an empty string

Reply:
you mean on `TestConnectionStringValidator_IsValid`? added